src/process/volcanic/CCPr_Volcanic_Mod.F90

```diff
@@ -0,0 +1,381 @@
+!> \brief CCPR Volcanice state types
+!!
+!! \defgroup catchem_Volcanic_process
+!!
+!! \author Lacey Holland and Wei Li
+!! \date 10/2024
+!!!>
+MODULE CCPR_Volcanic_mod
+   USE Precision_mod
+   USE Error_Mod
+   USE DiagState_Mod, Only : DiagStateType
+   USE MetState_Mod,  Only : MetStateType
+   USE ChemState_Mod, Only : ChemStateType
+   USE Config_Opt_Mod, Only : ConfigType
+   USE EmisState_Mod, Only: EmisStateType
+
+   IMPLICIT NONE
+
+   PRIVATE
+
+   PUBLIC :: CCPR_Volcanic_Init
+   PUBLIC :: CCPR_Volcanic_Run
+   PUBLIC :: CCPR_Volcanic_Finalize
+   PUBLIC :: VolcanicStateType
+
+
+   !> \brief VolcanicStateType
+   !!
+   !! VolcanicStateType is the process-specific derived type.
+   !!
+   !! \param Activate Activate Process (True/False)
+   !! \param Scheme Scheme Option
+   !! \param nVolcanicSpecies # of Volcanic species
+   !! \param VolcanicSpeciesIndex Index of Volcanic species
+   !! \param VolcanicSpeciesName Name of Volcanic species
+   !! \param SpcIDs CATChem species IDs
+   !! \param CatIndex Index of emission category in EmisState
+   !! \param TotalEmission Total emission of all species at each level [kg/m^2/s]
+   !! \param EmissionPerSpecies Emission per species at each level [kg/m^2/s]
+   !! \param FileDir Input file directory for reading in emissions
+   !!
```

Comment:
Let's update this group of comments to include more spacing between the parameter name and description. Also Scheme looks to actually be SchemeOpt below.